app/web/src/components/KeyValue/KeyValue.tsx

```diff
@@ -0,0 +1,57 @@
+import Svg from 'src/components/Svg/Svg'
+
+interface KeyValueType {
+  keyName: string
+  children: React.ReactNode
+  hide?: boolean
+  canEdit?: boolean
+  onEdit?: () => void
+  isEditable?: boolean
+  bottom?: boolean
+  className?: string
+}
+
+const KeyValue = ({
+  keyName,
+  children,
+  hide = false,
+  canEdit = false,
+  onEdit,
+  isEditable = false,
+  bottom = false,
+  className = '',
+}: KeyValueType) => {
+  if (!children || hide) return null
+  return (
+    <div className={'flex flex-col ' + className}>
+      <div
+        className={
+          'text-ch-blue-400 font-fira-code flex items-center leading-4 text-sm whitespace-nowrap ' +
+          (bottom ? 'order-2' : '')
```

Comment:
I'm not requesting a change this is just an observation. I noticed in the [headlessui docs](https://headlessui.dev/react/menu#basic-example) they follow a pattern like this for condition classes
```
className={`${active && 'bg-blue-500'}`}
```
At first I didn't like it because the result will often be `className="false"` so even `className="false false"` if you have more of them, but I guess there's no harm in having classes that don't do anything in there

Reply:
Oh interesting, yeah that's what I was avoiding here, but it would be shorter code their way. I guess the `false`s would have a negligible impact on things really.